openapi/resource-server.yaml

```diff
@@ -1012,6 +1019,125 @@ components:
         - sentAmount
         - createdAt
         - updatedAt
+    outgoing-payment-with-spent-amounts:
+      title: Outgoing Payment With Grant Spent Amounts
+      description: 'An **outgoing payment** resource represents a payment that will be, is currently being, or has previously been, sent from the wallet address.'
+      type: object
+      examples:
```

Comment:
Do we need both of these examples?

Reply:
I removed the second example.

For context, I followed the 2 examples in the original `outgoing-payment`. But I don't necessarily think we need 2 examples.